src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --default-color-link: #000;
+  --color-active-link: #00acdc;
+  --background-header: #fff;
+  --box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
According to the checklist, remember to use fallback fonts - alternative font-family in case the main one doesn't work. You can add a generic font family such as 'sans-serif' as a fallback.